### PR TITLE
feat: update production config on feature branch

### PR DESCRIPTION
**Pull request by thegentlemanphysicist**

### CODE REVIEW

File: transition-scripts/values/c6af30/dev/values-golddr-c6af30-dev-standby.yaml [hunk 1]
```diff
@@ -1,19 +1,22 @@
 replicaCount: 0
 
-resources:
```
Comment (jlangy) [inline transition-scripts/values/c6af30/dev/values-golddr-c6af30-dev-standby.yaml:3]: Is it intentional to remove the resource block for this env?

File: transition-scripts/values/c6af30/dev/values-gold-c6af30-dev-active.yaml [hunk 2]
```diff
@@ -1,15 +1,36 @@
...
+  - name: QUARKUS_LOG_FILE_ROTATION_FILE_SUFFIX
+    value: .zip
+  # There are no vanity urls in c6
+  - name: KC_HOSTNAME_URL
+    value: https://sso-keycloak-c6af30-dev.apps.gold.devops.gov.bc.ca/auth/
+  - name: KC_HOSTNAME_ADMIN_URL
+    value: https://sso-keycloak-c6af30-dev.apps.gold.devops.gov.bc.ca/auth/
+
+
+resources:
```
Comment (jlangy) [inline transition-scripts/values/c6af30/dev/values-gold-c6af30-dev-active.yaml:22]: The memory constraint is good to be the same for all envs eh?

**Review by jlangy — COMMENTED** [review]
Looks good to me, just left two questions

File: transition-scripts/values/c6af30/dev/values-gold-c6af30-dev-active.yaml (hunk 2 above)
Comment (thegentlemanphysicist) [reply transition-scripts/values/c6af30/dev/values-gold-c6af30-dev-active.yaml:22]: Yeah they didn't need to ge modified from what was already there

**Pull request merged**